exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/BaseRepeatedValueVector.java

```diff
@@ -123,6 +123,15 @@ public int getBufferSize() {
   }
 
   @Override
+  public int getBufferSizeFor(int valueCount) {
+    if (getAccessor().getValueCount() == 0) {
+      return 0;
+    }
+
+    return offsets.getBufferSizeFor(valueCount) + vector.getBufferSizeFor(valueCount);
```

Comment:
Based on setInitialValueCapacity(), I think you're right. Will do. (But I'm not going to go back and patch the EBF for that, because it'll just be off by one offsets value, or 4 bytes, which isn't going to kill the emergency fix.)